openpype/hosts/houdini/plugins/publish/collect_mantra_rop.py

```diff
@@ -31,49 +31,65 @@ class CollectMantraROPRenderProducts(pyblish.api.InstancePlugin):
     def process(self, instance):
 
         rop = hou.node(instance.data.get("instance_node"))
+        frame_start = instance.data["frameStart"]
+        frame_end = instance.data["frameEnd"]
 
         # Collect chunkSize
         chunk_size_parm = rop.parm("chunkSize")
-        if chunk_size_parm:
-            chunk_size = int(chunk_size_parm.eval())
-            instance.data["chunkSize"] = chunk_size
-            self.log.debug("Chunk Size: %s" % chunk_size)
-
-            default_prefix = evalParmNoFrame(rop, "vm_picture")
-            render_products = []
-
-            # Default beauty AOV
-            beauty_product = self.get_render_product_name(
-                prefix=default_prefix, suffix=None
+        if not chunk_size_parm:
+            return
+
+        chunk_size = int(chunk_size_parm.eval())
+        instance.data["chunkSize"] = chunk_size
+        self.log.debug("Chunk Size: {}".format(chunk_size))
+
+        default_prefix = evalParmNoFrame(rop, "vm_picture")
+        render_products = []
+
+        # Default beauty AOV
+        beauty_product = self.get_render_product_name(
+            prefix=default_prefix, suffix=None
+        )
+        render_products.append(beauty_product)
+
+        files_by_aov = {
+            "beauty": expected_files.generate_expected_filepaths(
+                frame_start, frame_end, beauty_product)
+        }
+
+        # get the number of AOVs
+        aov_numbers = rop.evalParm("vm_numaux")
+        if aov_numbers < 0:
+            return
+
+        # get the filenames of the AOVs
+        for index in range(1, aov_numbers + 1):
+            var_ = rop.evalParm("vm_variable_plane{:>1}".format(index))
+
+            # skip empty variables
+            if not var_:
+                continue
+
+            aov_name = "vm_filename_plane{:>1}".format(index)
+            aov_boolean = "vm_usefile_plane{:>1}".format(index)
+            aov_enabled = rop.evalParm(aov_boolean)
+            has_aov_path = rop.evalParm(aov_name)
+
+            # skip disabled AOVs
+            if not (has_aov_path and aov_enabled == 1):
+                continue
```

Comment:
Why are you formatting using `{:>1}`? Formatting with `{}` provides the same output for this.

```suggestion
            # skip empty variables
            if not rop.evalParm("vm_variable_plane{}".format(index)):
                continue

            # skip disabled AOVs
            aov_name = "vm_filename_plane{}".format(index)
            if rop.evalParm(aov_name) != 1:
                continue
                
            # skip AOV without use file set
            has_aov_path = rop.evalParm("vm_usefile_plane{}".format(index))
            if not has_aov_path:
                continue
```

Reply:
added here fc42095d7e6bd37fd0e65bc4f2f50446f2d96378